app/views/document_lead_image/crop.html.erb

```diff
@@ -5,7 +5,9 @@
   <%= t("document_lead_image.crop.description") %>
 </p>
 
-<%= form_tag(crop_document_lead_image_path(@document, @image, next_screen: params[:next_screen]), method: :patch) do %>
+<%= form_tag(crop_document_lead_image_path(@document, @image, wizard: params[:wizard]), method: :patch) do %>
+  <%= hidden_field_tag :wizard, params[:wizard] %>
```

Comment:
Do we need this hidden field if it's in the query string?

Reply:
Ah, good spot - I was aiming for the hidden field to keep the method call short.